src/language/providers/utils.ts

```diff
@@ -251,46 +316,72 @@ export function checkSetVariableOpen(
   return false
 }
 
+export function getNsPrefix(
+  document: vscode.TextDocument,
+  position: vscode.Position
+) {
+  var nsPrefix = ''
+  var initialLineNum = position.line
+  var lineNum = 0
+  while (initialLineNum !== 0) {
```

Comment:
initialLineNum never changes, this isn't a good condition for breaking the loop. If a match is never found (e.g. and empty file) then this will be stuck in an infinite loop.